webapp/sources/rudder/rudder-web/src/main/elm/sources/SystemUpdateScore.elm

```diff
@@ -0,0 +1,84 @@
+port module SystemUpdateScore exposing (..)
+
+import Browser
+import Html
+import Html.String exposing (..)
+import Html.String.Attributes exposing (class, title)
+import Json.Decode exposing (..)
+import Json.Decode.Pipeline exposing (..)
+import String.Extra
+
+port sendHtml : String -> Cmd msg
+port getValue : (Value -> msg) -> Sub msg
+
+type Msg = NewScore Value
+
+type alias SystemUpdateStats =
+  { nbPackages : Int,
+    security : Maybe Int,
+    patch : Maybe Int,
+    defect : Maybe Int,
+    enhancement : Maybe Int,
+    other : Maybe Int
+  }
+
+decodeSystemUpdateStats : Decoder SystemUpdateStats
+decodeSystemUpdateStats =
+  succeed SystemUpdateStats
+    |> required "nbPackages" int
+    |> optional "security" (maybe int) Nothing
+    |> optional "updates" (maybe int) Nothing
+    |> optional "defect" (maybe int) Nothing
+    |> optional "enhancement" (maybe int) Nothing
+    |> optional "other" (maybe int) Nothing
+
+buildScoreDetails :  SystemUpdateStats  -> Html msg
+buildScoreDetails details =
+  let
+    toBadge : String -> String -> Maybe Int -> Html msg
+    toBadge id iconClass value =
+      case value of
+        Just v  ->
+          let
+            valueTxt = String.fromInt v
+            titleTxt = (String.Extra.humanize id) ++ ": " ++ valueTxt
+          in
+            span[class ("badge badge-" ++ id), title titleTxt][i[class ("fa fa-" ++ iconClass)][], text valueTxt]
+        Nothing -> text ""
+  in
+        div[]
+          [ toBadge "security"    "warning" details.security
+          , toBadge "bugfix"      "bug"     details.defect
+          , toBadge "enhancement" "plus"    details.enhancement
+          , toBadge "update"      "box"     details.patch
+          ]
+
+main =
+  Browser.element
+    { init = init
+    , view = always (Html.text "")
```

Comment:
Is it intended to not display anything ?

Reply:
Yep! these apps only exist to provide details for generic node details api that does not know how to handle them (in fact it could handle system-updates and compliance by itself but i wanted them to be like the plugins so that we know the recipe to make a plugin from Rudder